### PR TITLE
Add support for tag alias

### LINKED ISSUE #16: Add ability to specify tag in config
Hi, would be nice to be able to specify a tag during the build in the config so the image would be published with, for example, a branch-named tag or with `latest` tag without the suffix.

### PR DESCRIPTION
**Pull request by igrybkov**
Fixes #16

### CODE REVIEW

**Comment by igrybkov** [comment]
@mrsimpson I've tested it manually by copying openfaas function files into the directory and running `entrypoint.sh` with arguments. This way it worked for me.
The test `CI / test-simple` fails because either `DOCKER_USERNAME` or `DOCKER_PASSWORD` is empty. If I'm wrapping it into `if` with the check it works well, but I don't think this should be a part of the final code.
```
if [ ! -z $DOCKER_PASSWORD ]; then
    echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin $REGISTRY
fi
```

**Comment by igrybkov** [comment]
Hi @mrsimpson, is there any chance to get this merged?

**Comment by mrsimpson** [comment]
@igrybkov So sorry, I was off for some days and couldn't even have a glance at it. I'll try to get this Indonesia's today or tomorrow evening!

File: entrypoint.sh [hunk 1]
```diff
@@ -3,18 +3,19 @@
...
 
-# $1 stack-file
-# $2 docker-username
-# $3 docker-password
-# $4 platforms
-# $5 deploy
-# $6 gateway
-# $7 openfaas-username
-# $8 openfaas-password
+STACK_FILE="$1"
```
Comment (mrsimpson) [inline entrypoint.sh:6]: much better ... Obviously I'm not a frequent bash scripter ;)

**Review by mrsimpson — APPROVED** [review]
Thanks, @igrybkov for your contribution, quality improvement and finally patience

**Pull request merged**